optaplanner-persistence/optaplanner-persistence-jackson/src/main/java/org/optaplanner/persistence/jackson/api/score/PolymorphicScoreJacksonJsonSerializer.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
```

Comment:
Should be 2018 unless the file has already existed an was just modified & moved.